docs/source/usage/deployment.rst

```diff
@@ -158,7 +162,7 @@ This is where we take everything we've done up to this point and start the deplo
      9. For stack parameters, enter the values decided in :ref:`Getting Started` Step 2.
        * For service, enter your <service> name that you determined earlier.
        * For environment, choose “dev”, "staging", or "prod" according to the environment you are working out of.
-       * In the TOKEN_ARN field, you'll need to enter your token arn to correctly map the IAM permissions for this resource.
+       * In the SSMParamName field, you'll need to enter the name of the SSM parameter used to store the auth token, to correctly map the IAM permissions for this resource.
```

Comment:
the commas in this line aren't needed as the following is a complete thought.
"In the SSMParamName field you'll need to enter the name of the SSM parameter used to store the auth token to correctly map the IAM permissions for this resource."

Otherwise if we removed what was within the two commas, it would make no sense:
"In the SSMParamName field to correctly map the IAM permissions for this resource."
And the last piece on it's own is not a complete clause: "To correctly map the IAM permissions for this resource"

Reply:
Will fix, thanks.